server/src/test/java/org/apache/iotdb/db/utils/datastructure/TVListTest.java

```diff
@@ -26,8 +26,18 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-public class LongTVListTest {
+public class TVListTest {
 
+  @Test
+  public void testDoubleTVList1(){
+    TVList tvList = new DoubleTVList();
+    for (long i = 0; i < 1000; i++) {
+      double j = 1.2345678+Math.random();
+      tvList.putDouble(i, j);
+    }
+    tvList.sort();
+    tvList.getTimeValuePair(0);
+  }
```

Comment:
LGTM. Why not add a `testFloatTVList` as well?

Reply:
OK，add